packages/contracts/package.json

```diff
@@ -84,6 +92,10 @@
     "tmp-promise": "^3.0.3",
     "ts-node": "^8.1.0",
     "typechain": "^5.2.0",
-    "typescript": "^4.4.4"
+    "typescript": "^4.4.4",
+    "zksync-ethers": "^5.0"
+  },
+  "dependencies": {
+    "@matterlabs/hardhat-zksync-ethers": "0.0.1-beta.2"
```

Comment:
is there a reason this one must be a dependency? just curiuse

Reply:
nope. Fixed in https://github.com/aragon/osx/pull/596/commits/916994e110ba57384ee62ca6c3ef17c8870783a7